src/assets/code-hashes.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
nit: I would keep this in the `bin` assets. I don't know if we want to publish this with the package or not (my intuition says "no").

You can even embed it in the `bash` script:

```
JSON="$(cat <<JSON
{
  // ...
}
JSON
)"
```

Reply:
We will NOT inline the JSON (leads to an unruly and large bash file, which isn't nice). Instead, we will add `codeHash` field to each deployment JSON.